core/validation/initial_check.py

```diff
@@ -21,7 +21,7 @@ def extract_submission_details(
     :param reporting_round: Integer representing the round being ingested.
     :return: A dictionary containing the extracted submission details.
     """
-    details_dict = {"ValueChecks": {}, "NullChecks": {}}
+    details_dict = {"ValueChecks": {}}
```

Comment:
Can we just get rid of the nesting now and instantiate `value_checks = {}` and then return `value_checks`?

Reply:
Nothing obvious comes to mind for me but maybe there's a better name than `value_checks`..

Reply:
wrong_input_checks as that is the failure we're testing for